dev-docs/conventions.md

```diff
@@ -35,43 +35,41 @@ Further we try to adhere to the following guidelines:
   Example:
 
   ```Go
-  log.Infof("This is a log message")
-  log.Infof("Waiting to do something...")
+  log.Info("This is a log message")
+  log.Info("Waiting to do something...")
   log.Error("A critical error occurred!")
   ```
 
-* Use the `With()` method to add structured context to your log messages. The context tags should be easily searchable to allow for easy log filtering. Try to keep consistent tag naming!
+* Use the `WithAttrs()` method to add structured context to your log messages. The context tags should be easily searchable to allow for easy log filtering. Try to keep consistent tag naming!
 
   Example:
 
   ```Go
-  log.With(zap.Error(someError), zap.String("ip", "192.0.2.1")).Errorf("Connecting to IP failed")
+  log.WithAttrs(slog.Any("error" someError), slog.String("ip", "192.0.2.1")).Error("Connecting to IP failed")
```

Comment:
The idiomatic way seems to be
```go
log.Error("Connecting to IP failed", "error", someError, "ip", "192.0.2.1")
```
Any reason we should deviate from this? (Except for it's more similar to what we did before.)

Reply:
Not really, if this is preferred I can change it accordingly across the codebase.

Reply:
I'd like to have at least one more opinion on this before you invest time in changing this :)
(Another option would be to recommend the new style in conventions.md without adapting all the existing code.)

Reply:
I'm in favor of Thomas' suggested solution, while also proposing to not adapt this in this PR, but rather do a follow-up if desired.

Reply:
Let's adapt the recommendation in conventions.md, but not change all existing code. (At least not in this PR.)